math/physics/src/TLorentzVector.cxx

```diff
@@ -5,9 +5,22 @@
 //    Oct 20 1999: dito in Double_t operator()
 //    Jan 25 2000: implemented as (fP,fE) instead of (fX,fY,fZ,fE)
 
-
 /** \class TLorentzVector
     \ingroup Physics
+
+## Disclaimer
+In order to represent 4-vectors, TLorentzVector shall not be used.
+ROOT provides specialisations of the ROOT::Math::LorentzVector template which
+are superior from the runtime performance offered, i.e.:
+  - ROOT::Math::XYZTVector vector based on x,y,z,t coordinates (cartesian) in double precision
```

Comment:
The "XYZ" specializations should probably be last in this enumeration; eta/phi or momentum-based LVs are much more common.